rpc/tests/rpc_command_tests/get_blocks_in_chain.rs

```diff
@@ -16,7 +17,164 @@ async fn get_blocks_in_chain_ok() {
     assert!(blocks.len() > 0);
 }
 
+#[tokio::test]
+async fn get_blocks_with_optional_params_bad_head_hash_fail() {
+    let bad_head_hash = "".to_string();
+    let command = generate_get_blocks_command_with_explicit_params(None, Some(bad_head_hash), None);
+    let client = get_rpc_client();
+    assert!(client.check_node_online().await);
+
+    let client_response = client.execute(&command).await;
+    assert!(client_response.is_err());
+}
+
+#[tokio::test]
+async fn get_blocks_from_head_optional_params_ok() {
+    let head_hash = "BKmXPAiniarmJAxvv3T3CQ9sa42TbLiDdE6c57Gc74NjcKNFQBE".to_string();
```

Comment:
Maybe we should have this be a constant for all rpc command tests that use a head hash